src/network/CMakeLists.txt

```diff
@@ -4,45 +4,88 @@
 cmake_minimum_required(VERSION 3.1.0)
 
 project(realsense2-net VERSION 1.0.0 LANGUAGES CXX C)
+set(THREADS_PREFER_PTHREAD_FLAG ON)
+find_package(Threads REQUIRED)
 
 # Save the command line compile commands in the build output
 set(CMAKE_EXPORT_COMPILE_COMMANDS 1)
 
-add_definitions(-DNEWLOCALE_NOT_USED=1 -DBSD=1 -DSOCKLEN_T=socklen_t -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE=1 -DALLOW_RTSP_SERVER_PORT_REUSE=1 -DNO_OPENSSL=1)
+add_definitions(-D_WINSOCKAPI_ -DNEWLOCALE_NOT_USED=1 -DBSD=1 -DSOCKLEN_T=socklen_t -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE=1 -DALLOW_RTSP_SERVER_PORT_REUSE=1 -DNO_OPENSSL=1 -DRESPONSE_BUFFER_SIZE=100000)
```

Comment:
D_WINSOCKAPI_  is WinOS only - shall be encapsulated properly

Reply:
Will be fixed.